.github/workflows/ci.yml

```diff
@@ -195,7 +195,16 @@ jobs:
           cargo fmt -- --check
           cargo clippy --all-targets --all-features --all
 
-      - name: Test
+      - name: Install cargo-tarpaulin
+        if: matrix.rust == 'stable'
+        run: which cargo-tarpaulin || cargo install cargo-tarpaulin
```

Comment:
This workaround is still needed on CI? I think recent cargo doesn't require it anymore.

Reply:
It failed for me running the CI twice consecutively yesterday on my test project to debug the 0% coverage https://github.com/xd009642/crates-io-test/actions

Reply:
Hm, that's interesting, I guess cargo recognizes it as an untracked file since it's introduced by cache (or another reason) 🤔. Thanks for clarifying, then I'm fine as-is.

Reply:
Wouldn't it be reasonable to cache `tarpaulin`? There is no need to rebuild it CI run I feel?

https://github.com/marketplace/actions/cache#skipping-steps-based-on-cache-hit
https://github.com/actions/cache/blob/main/examples.md#rust---cargo

Reply:
AFAICT https://github.com/rust-lang/crates.io/blob/2b3bdf116729f34b8d767199018beb5efc5ce3c1/.github/workflows/ci.yml#L119-L123 should already cache the binaries including `diesel-cli` and `cargo-tarpaulin`

Reply:
Ahh, sorry then. Must have overlooked that part :/